Idref.js

```diff
@@ -0,0 +1,83 @@
+{
+	"translatorID": "271ee1a5-da86-465b-b3a5-eafe7bd3c156",
+	"label": "Idref",
+	"creator": "Sylvain Machefert",
+	"target": "^https?://www\\.idref\\.fr/.*",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "g",
+	"lastUpdated": "2018-02-03 09:39:41"
+}
+
+function detectWeb(doc, url) { 
+	if (ZU.xpathText(doc, '//div[@class="detail_bloc_biblio"]')) {
+		return "multiple";
+	}
+	
+}
+
+function doWeb(doc, url)
+{
+	if (detectWeb(doc, url) == "multiple") {
+		var resultsTitle = ZU.xpath(doc, '//div[@id="perenne-references-docs"]/span[contains(@class, "detail_value")]');
+		var resultsHref = ZU.xpath(doc, '//div[@id="perenne-references-docs"]/span[contains(@class, "detail_label")]/a/@href');
+		
+		items = {};
+		for (var i in resultsTitle) {
```

Comment:
iterate through arrays using either `for (let i=0; i<resultsTitle.lenght; i++)` or `for (result of resultsTitle)`